lg_volume_control/src/lg_volume_control/volume_control.py

```diff
@@ -0,0 +1,50 @@
+import rospy
+import commands
```

Comment:
Not requesting a change for this, but `commands` has been [deprecated since Python 2.6](https://docs.python.org/2/library/commands.html).